app/controllers/staff/activity_forms_controller.rb

```diff
@@ -1,3 +1,5 @@
+require "#{Rails.root}/vendor/data/country_to_region_mapping.rb"
```

Comment:
I guess yaml is consistent with the rest of the vendor data, but if you are saying there is a performance benefit I wonder why we chose yaml to begin with?

Reply:
The YAML files we use have multiple things in each data "chunk" (e.g. code, description, name) so a one-to-one mapping with a hash would make less sense. I think!?